src/quo2/components/drawers/drawer_buttons/style.cljs

```diff
@@ -0,0 +1,49 @@
+(ns quo2.components.drawers.drawer-buttons.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def outer-container {:height 216})
+
+(def top-card
+  {:display            :flex
+   :height             "100%"
```

Comment:
redundant :height "100%"

Reply:
I commented about this above. So without this the card disappears, so it's not redundant. Although perhaps I should use a different approach? 🤔

Reply:
something wrong yet. you should never specify display :flex and height "100%", you should just use `:flex 1` instead

Reply:
I've fixed some of these problems by using other flex properties, like `flex` and `flex-grow`. It shouldn't be needed as @flexsurfer said.

Reply:
cool, will adjust 👍